rust/src/blocks/cb_csv.rs

```diff
@@ -58,6 +68,12 @@ impl Block for CSVRead {
     "CSV.Read"
   }
 
+  fn help(&mut self) -> OptionalString {
```

Comment:
Nice!

Though we have other "help" functions: `inputHelp()`and `outputHelp()` 😉